kpi/tests/api/test_api_environment.py

```diff
@@ -156,7 +172,7 @@ def test_mfa_per_user_availability_while_globally_enabled_as_anonymous(self):
     def test_social_apps(self):
         # GET mutates state, call it first to test num queries later
         self.client.get(self.url, format='json')
-        queries = 19
+        queries = 20
```

Comment:
This query is being executed twice for some reason:
```sql
SELECT "constance_config"."id", "constance_config"."key", "constance_config"."value" FROM "constance_config" WHERE "constance_config"."key" = 'MFA_LOCALIZED_HELP_TEXT' LIMIT 21
```
I had to give up caring about this for now. I have another branch that I'm about to PR anyway that reduces (or at least was reducing) the number of queries from 19 to 18, so… :shrug:

Reply:
https://github.com/kobotoolbox/kpi/pull/4550/files#r1271244110